zipline/finance/performance/tracker.py

```diff
@@ -413,6 +408,22 @@ def check_upcoming_dividends(self, completed_date):
             # notify periods to update their stats
             period.handle_dividends_paid(net_cash_payment)
 
+    def check_asset_auto_closes(self, next_trading_day):
+        """
+        Check if we currently own any futures whose notification_date is
```

Comment:
Sometimes we use the term notice date, and sometimes notification date. Let's stick to notice date, since it's shorter and is used on CME's website.